pkg/api/bzz.go

```diff
@@ -128,10 +128,14 @@ func (s *Service) fileUploadHandler(logger log.Logger, w http.ResponseWriter, r
 		queries.FileName = fr.String()
 	}
 	if err := s.validate.Var(queries.FileName, "startsnotwith=/"); err != nil {
-		// TODO: return standard validation error.
+		valErr := &validationError{
+			Entry: "queries.FileName",
+			Value: queries.FileName,
+			Cause: err,
+		}
 		logger.Debug("bzz upload file: / in prefix not allowed", "file_name", queries.FileName, "error", err)
 		logger.Error(nil, "bzz upload file: / in prefix not allowed", "file_name", queries.FileName)
-		jsonhttp.BadRequest(w, "/ in prefix not allowed")
+		jsonhttp.BadRequest(w, valErr)
```

Comment:
maybe for `bad request` we should not just pass on the technical error but use a plain string instead

Reply:
If there is a validation error, if would write a resp like this
```
got json response "{\"code\":400,\"message\":\"invalid query params\",\"reasons\":[{\"field\":\"name\",\"error\":\"want startsnotwith:/\"}]}"
```